lib/components/primitive-components/Via.ts

```diff
@@ -53,7 +53,6 @@ export class Via extends PrimitiveComponent<typeof viaProps> {
       x: newCenter.x,
       y: newCenter.y,
     })
-    this.matchedPort?._setPositionFromLayout(newCenter)
```

Comment:
This is kind of weird, just noting it

Reply:
I guess a via never has a matched port

Reply:
Yeah, that's why removed it. By mistake I added it in my last PR